<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vlcPlayerClass</class>
 <widget class="QWidget" name="vlcPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vlcPlayer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame
{
	background: #212426;
	border-radius:6px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_top" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>56</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>56</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget_top
{
	background-color: #1B1A2C;
	border-top-left-radius:6px;
	border-top-right-radius:6px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>14</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>14</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_logo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>56</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>56</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>华文彩云</family>
             <pointsize>22</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>VLC播放器</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>635</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_net" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	border:none;
	background-color:transparent;
	font-size: 16px;
	font-family: 微软雅黑;
	color: #01EEC3;
}

QPushButton:hover
{
	border-radius:10px;
	background-color: rgba(255, 255, 255, 0.2);
	font-size: 16px;
	font-family: 微软雅黑;
	color: #01EEC3;
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_topRight" native="true">
           <property name="styleSheet">
            <string notr="true">#pushButton_min
{
	border-image: url(:/WAAudioPanel/WAAudioPanel/ico_min.png);
}

#pushButton_max
{
	border-image: url(:/WAAudioPanel/WAAudioPanel/ico_max.png);
}

#pushButton_close
{
	border-image: url(:/WAAudioPanel/WAAudioPanel/ico_gb.png);
}

#pushButton_service
{
	border-image: url(:/WAAudioPanel/WAAudioPanel/ico_kf.png);
}

/*#pushButton_min:hover
{
	background-color: rgb(85, 162, 228);
}

#pushButton_max:hover
{
	background-color: rgb(85, 162, 228);
}

#pushButton_close:hover
{
	background-color: rgb(255, 84, 57);
}*/</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_min">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>最小化</string>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/qrc/qrc/ico_min.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_max">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>最大化</string>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/qrc/qrc/ico_max.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_close">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>关闭</string>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/qrc/qrc/ico_gb.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_video" native="true">
        <property name="styleSheet">
         <string notr="true">#widget_video
{
	border-bottom-left-radius:6px;
	border-bottom-right-radius:6px;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_player" native="true">
           <property name="styleSheet">
            <string notr="true">#widget_player
{
	background-color: rgb(0, 0, 0);
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_title" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#widget_title
{
	background-color: #1B1A2C;
	border-bottom-left-radius:6px;
	border-bottom-right-radius:6px;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_title_top" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>16</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_videoName">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">	font-size: 16px;
	font-family: 微软雅黑;
	color: #01EEC3;</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>839</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_title_main" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="vlcPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
